spec/integration/ClientSpec.php

```diff
@@ -29,7 +29,6 @@ function let(){
 
         $this->beConstructedWith([
             'baseUrl'       => getenv('NOTIFY_API_URL'),
-            'serviceId'     => getenv('SERVICE_ID'),
```

Comment:
Thanks, just wanted to make sure that it's tested even if the param is optional - found the unit test ;)